dm/config/checking_item.go

```diff
@@ -42,7 +42,7 @@ const (
 	LightningRegionDistributionChecking = "region_distribution"
 	LightningDownstreamVersionChecking  = "downstream_version"
 	LightningFreeSpaceChecking          = "free_space"
-	LightningSortingSpaceChecking       = "enough_sorting_space"
```

Comment:
not needed?

Reply:
seem duplicate with `free_space`

Reply:
"sorting space" is a property of DM worker, but precheck is done at DM master. I forget this limit